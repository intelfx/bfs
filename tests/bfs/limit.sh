bfs_diff -s basic -type f -print -limit 4
