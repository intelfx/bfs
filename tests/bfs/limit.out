basic/a
basic/b
basic/c/d
basic/e/f
